tools/perf/ucc_pt_coll_reduce_scatter.cc

```diff
@@ -0,0 +1,92 @@
+#include "ucc_pt_coll.h"
+#include "ucc_perftest.h"
+#include <ucc/api/ucc.h>
+#include <utils/ucc_math.h>
+#include <utils/ucc_coll_utils.h>
+
+ucc_pt_coll_reduce_scatter::ucc_pt_coll_reduce_scatter(ucc_datatype_t dt,
+                        ucc_memory_type mt, ucc_reduction_op_t op,
+                        bool is_inplace,
+                        ucc_pt_comm *communicator) : ucc_pt_coll(communicator)
+{
+    has_inplace_   = true;
+    has_reduction_ = true;
+    has_range_     = true;
+    has_bw_        = true;
+
+    coll_args.coll_type = UCC_COLL_TYPE_REDUCE_SCATTER;
+    coll_args.mask = 0;
+    if (is_inplace) {
```

Comment:
1 question: last time we discussed INPLACE for RS in WG we agreed that there can be 2 modes (result in the beginning of the buffer or with the offset). Which inplace is it? What is out default mode in UCC?